clients/instance/ibm-pi-helper.go

```diff
@@ -21,5 +29,9 @@ func NewIBMPIClient(ctx context.Context, sess *ibmpisession.IBMPISession, cloudI
 		session:         sess,
 		cloudInstanceID: cloudInstanceID,
 		ctx:             ctx,
+
+		volumeRequest:    sess.Power.PCloudVolumes,
```

Comment:
I do not understand why are we storing all these client objects in memory? It is already available in the sess attribute. You are just increasing the memory consumption and lines of code.